int32    event
float32    frequency
int32    number_killed
int32    number_created
int32    time_stamp
int32    life 